### ISSUE TITLE
Test failure: JIT\IL_Conformance\Old\Conformance_Base\ldarg_i\ldarg_i.cmd

### ISSUE THREAD

**Issue by BruceForstall** [dotnet/runtime#79829]
Looks like it's failing in all configurations.

```
    JIT\IL_Conformance\Old\Conformance_Base\ldarg_i\ldarg_i.cmd [FAIL]
      
      Return code:      1
      Raw output file:      C:\h\w\C55C0A68\w\B61A0982\uploads\Reports\JIT.IL_Conformance\Old\Conformance_Base\ldarg_i\ldarg_i.output.txt
      Raw output:
      BEGIN EXECUTION
       "C:\h\w\C55C0A68\p\corerun.exe" -p "System.Reflection.Metadata.MetadataUpdater.IsSupported=false"  ldarg_i.dll 
      Expected: 100
      Actual: 0
      END EXECUTION - FAILED
      FAILED
      Test Harness Exitcode is : 1
      To run the test:
      > set CORE_ROOT=C:\h\w\C55C0A68\p
      > C:\h\w\C55C0A68\w\B61A0982\e\JIT\IL_Conformance\Old\Conformance_Base\ldarg_i\ldarg_i.cmd
      Expected: True
      Actual:   False
      Stack Trace:
           at JIT_IL_Conformance._Old_Conformance_Base_ldarg_i_ldarg_i_._Old_Conformance_Base_ldarg_i_ldarg_i_cmd()
           at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
           at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

e.g., outerloop: https://dev.azure.com/dnceng-public/public/_build/results?buildId=115813&view=ms.vss-test-web.build-test-results-tab

**Comment by BruceForstall** [comment]
It looks like this test was changed with https://github.com/dotnet/runtime/pull/79705.

@janvorli Should we revert the test change? Disable the test temporarily until the issue is fixed?
cc @jakobbotsch

**Comment by janvorli** [comment]
I am sorry, I've only tested the modified tests when running in an unloadable context (where is passed). I have a fix ready.

**Comment by janvorli** [comment]
Fixed by #79834
